### PR TITLE
Add psremoting enable/disable cmdlet warning messages

### PR DESCRIPTION
**Pull request by PaulHigin**
# PR Summary

Adds warning messages to the Enable-PSRemoting and Disable-PSRemoting cmdlets that informs the user that the cmdlets only affect PSCore6 remoting endpoint configurations and not Windows PowerShell configurations.

## PR Context

As documented, Enable-PSRemoting and Disable-PSRemoting affects **all** PowerShell remoting on a Windows system, and Windows users will likely expect the cmdlets to continue working this way in PSCore6.  However, these cmdlets currently affect only PSCore6 remoting endpoints when run in PSCore6 and have no effect on Windows remoting endpoints.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [x] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [x] Issue filed: 8469
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: src/System.Management.Automation/resources/RemotingErrorIdStrings.resx [hunk 1]
```diff
@@ -1686,4 +1686,11 @@ All WinRM sessions connected to PowerShell session configurations, such as Micro
...
   </data>
-</root>
+  <data name="PSCoreRemotingDisableWarning" xml:space="preserve">
+    <value>PowerShell remoting has been disabled only for PowerShell Core configurations and does not affect Windows PowerShell remoting configurations. Run this cmdlet in Windows PowerShell to affect all PowerShell remoting configurations.
+</value>
+  </data>
+  <data name="PSCoreRemotingEnableWarning" xml:space="preserve">
+    <value>PowerShell remoting has been enabled only for PowerShell Core configurations and does not affect Windows PowerShell remoting configurations. Run this cmdlet in Windows PowerShell to affect all PowerShell remoting configurations.</value>
+  </data>
+</root>
```
Comment (TravisEz13) [inline src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]: missing newline at end of file

File: src/System.Management.Automation/resources/RemotingErrorIdStrings.resx [hunk 2]
```diff
@@ -26,41 +26,41 @@
...
 
     mimetype: application/x-microsoft.net.object.bytearray.base64
-    value   : The object must be serialized into a byte array
+    value   : The object must be serialized into a byte array 
             : using a System.ComponentModel.TypeConverter
             : and then encoded with base64 encoding.
     -->
-  <xsd:schema id="root" xmlns="" xmlns:xsd="https://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
-    <xsd:import namespace="https://www.w3.org/XML/1998/namespace" />
+  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
```
Comment (TravisEz13) [inline src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]: Please don't revert to http

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/resources/RemotingErrorIdStrings.resx (hunk 2 above)
Comment (SteveL-MSFT) [reply src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]: As far as I can tell, the official URI for the xsd xmlns is `http://www.w3.org/2001/XMLSchema`.  This is a URI and not a URL, so the https matters in that it is incorrect.  It seems we need to fix this in all our resx files as it is technically wrong when we bulk updated the http to https.

Comment (SteveL-MSFT) [reply src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]: The bulk change should be in a separate PR.

**Review by SteveL-MSFT — APPROVED** [review]

File: src/System.Management.Automation/resources/RemotingErrorIdStrings.resx (hunk 1 above)
Comment (PaulHigin) [reply src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]: This is intended.  No newline is needed in this case.

File: src/System.Management.Automation/resources/RemotingErrorIdStrings.resx [hunk 3]
```diff
@@ -26,40 +26,40 @@
...
             : and then encoded with base64 encoding.
 
     mimetype: application/x-microsoft.net.object.bytearray.base64
-    value   : The object must be serialized into a byte array
+    value   : The object must be serialized into a byte array 
             : using a System.ComponentModel.TypeConverter
             : and then encoded with base64 encoding.
     -->
-  <xsd:schema id="root" xmlns="" xmlns:xsd="https://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
+  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
```
Comment (TravisEz13) [inline src/System.Management.Automation/resources/RemotingErrorIdStrings.resx]:
```suggestion
  <xsd:schema id="root" xmlns="" xmlns:xsd="https://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
```

**Review by TravisEz13 — APPROVED** [review]

**Comment by PaulHigin** [comment]
@adityapatwardhan   Can this be merged?  Thanks!

**Pull request merged**